src/components/Label/Label.md

```diff
@@ -0,0 +1,35 @@
+# Label
+Used to label form elements, such as TextField and Dropdown.
```

Comment:
Removed the comma. I don't think we need the full list here, just a couple examples of where it's used.